### PR TITLE
{lib}[GCCcore/12.2.0] nf-core v2.10, psycopg2 v2.9.9, pydantic v1.10.13, ...

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

This is basically the [nf-core tools package](https://nf-co.re/tools) which are useful for running [nf-core ](https://nf-co.re) pipelines (or preparing them, e.g. downloading the required containers)

I found a module loading a conda env on another HPC system and figured this would be useful to have in EB especially with that large amount of dependencies

### CODE REVIEW

**Comment by Flamefire** [comment]
@smoretti @SIB-software This might be interesting for you too as you added the `nf-core-mag` EC.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 24 out of 33 (5 easyconfigs in total)
taurusi6605.taurus.hrsk.tu-dresden.de - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/Flamefire/d75828eeb87727e8d3c223332a8cf7a4 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
taurusi6605.taurus.hrsk.tu-dresden.de - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/Flamefire/aebb1a3f4507e0d8d49f809e05441705 for a full test report.

**Comment by Flamefire** [comment]
@boegel Any chance to get this in? Especially as it adds a couple of ECs of Python modules which can be useful for other ECs (or are updates to existing ones)

File: easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ed4f41f6daecbeeb96e576ce414c41d2d876daa9a16cb35fa8ed8c2ddfad0224']
+
+builddependencies = [('binutils', '2.39')]
+dependencies = [
+    ('Python', '3.10.8'),
+]
```
Comment (Flamefire) [inline easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-12.2.0.eb:20]:
```suggestion
dependencies = [('Python', '3.10.8')]
```

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
Going in, thanks @Flamefire!

**Pull request merged**